### PR TITLE
Some Readme improvements

### PR DESCRIPTION
**Pull request by helfio**
Thank you for your help on #9 
These are the steps I followed in order to setup my server. 
Please, let me know in case you think something should be different.

Cheers,
Pablo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,31 +6,51 @@ and like MM is shortcut, this container is a simillar shortcut that saves time.
...
 ```
-  docker-compose up --detach
+  $ sudo docker-compose up --detach
 ```
 ## Rebuild
 
 If you change app-config.js or or something in .env then you have to rebuild the image.
 ```
-  docker-compose up --build --detach
+  $ sudo docker-compose up --build --detach
```
Comment (misi) [inline README.md:53]: Depends of your settings user could be part of docker group  and then sudo not needed

File: README.md [hunk 2]
```diff
@@ -6,31 +6,51 @@ and like MM is shortcut, this container is a simillar shortcut that saves time.
...
-./run.sh
+$ sudo ./run.sh
 ```
 
-2. or with `docker-compose` 
+2. Or with `docker-compose` 
 / [install docker compose](https://docs.docker.com/compose/install/) /
 ```
-  docker-compose up --detach
+  $ sudo docker-compose up --detach
```
Comment (misi) [inline README.md:47]: Depends of your settings. User could be part of docker group and then sudo is not needed

Comment (helfio) [reply README.md:47]: Yes, you are right. Anyway, sudo works in both cases. I can remove it if you want.

Comment (misi) [reply README.md:47]: I am ok with it..

**Pull request merged**